n2survey/lime/survey.py

```diff
@@ -1476,4 +1438,4 @@ def export_to_file(
 
         new_data.to_csv(directory)
 
-        print("\nData exported!")
```

Comment:
@fudw I found this commit to be the one where many empty lines are removed, but I'm not sure how to revert to this commit, could you show me how please?

Reply:
Try this: https://stackoverflow.com/questions/4114095/how-do-i-revert-a-git-repository-to-a-previous-commit